### ISSUE TITLE
Line overflows are hidden

### ISSUE THREAD

**Issue by Mc-Zen** [Dherse/codly#5]
Hi, thanks for the awesome package!

I noticed that when a line exceeds the available width, it overflows in an unfavourable fashion. 

`````typ
#import "@preview/codly:0.1.0": *
#show: codly-init

#codly(
  languages: (
    typ: (name: "typ", icon: none, color: rgb("#239DAE")),
  ),
  // stroke-width: none
)

```
A really really really really really really really really really really really long line
```

\

```
A really really really really really really really really really really really long line
THe next line draws over the first one
```
`````

![image](https://github.com/Dherse/codly/assets/52877387/1bcacccf-9c29-4eec-a919-7b7b59371e66)

**Comment by Dherse** [comment]
Fixed in 1909b1a and will be in the upcoming 0.2.0 release ;)

**Comment by Mc-Zen** [comment]
Amazing!
